x-pack/plugins/elastic_assistant/server/routes/knowledge_base/get_knowledge_base_status.ts

```diff
@@ -54,38 +49,24 @@ export const getKnowledgeBaseStatusRoute = (
         const ctx = await context.resolve(['core', 'elasticAssistant', 'licensing']);
         const assistantContext = ctx.elasticAssistant;
         const logger = ctx.elasticAssistant.logger;
-        const telemetry = assistantContext.telemetry;
 
         try {
           // Use asInternalUser
-          const esClient = (await context.core).elasticsearch.client.asInternalUser;
-          const elserId = await getElser();
           const kbResource = getKbResource(request);
 
           // FF Check for V2 KB
           const v2KnowledgeBaseEnabled = isV2KnowledgeBaseEnabled({ context: ctx, request });
 
-          const kbDataClient = await assistantContext.getAIAssistantKnowledgeBaseDataClient(
-            v2KnowledgeBaseEnabled
-          );
+          const kbDataClient = await assistantContext.getAIAssistantKnowledgeBaseDataClient({
+            v2KnowledgeBaseEnabled,
+          });
           if (!kbDataClient) {
             return response.custom({ body: { success: false }, statusCode: 500 });
           }
 
-          // Use old status checks by overriding esStore to use kbDataClient
-          const esStore = new ElasticsearchStore(
-            esClient,
-            kbDataClient.indexTemplateAndPattern.alias,
-            logger,
-            telemetry,
-            elserId,
-            kbResource,
-            kbDataClient
-          );
-
-          const indexExists = await esStore.indexExists();
-          const pipelineExists = await esStore.pipelineExists();
-          const modelExists = await esStore.isModelInstalled(elserId);
+          const indexExists = true; // Installed at startup, always true
+          const pipelineExists = true; // Installed at startup, always true
```

Comment:
Here shortly we should update this API to remove these fields. Would be nice to return `modelId` as well in case the client ends up needing it.